massa-client/src/cmds.rs

```diff
@@ -439,7 +439,7 @@ impl Command {
     ///     it means that we don't want to print anything we just want the json output
     pub(crate) async fn run(
         &self,
-        client: &Client,
+        client: &mut Client,
```

Comment:
Why the client become a mutable reference?

Reply:
Because MassaServiceClient<T>::get_version is using a &mut self

Reply:
For now it's not called anywhere. It will be in the next PR (to include Versioning info). You can have a look at the initial draft PR: https://github.com/massalabs/massa/pull/3953 here.

Reply:
Mhh ok it's better to not include changes for an other PR but it's ok